Gemfile

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+source "https://rubygems.org"
+
+# Specify your gem's dependencies in rubocop-mdsol.gemspec
+gemspec
+
+gem "rake", "~> 13.0"
```

Comment:
add this as a dev dep and do without the Gemfile entirely?

Reply:
Ah right! Fixed in 8415f8d